src/ansys/aedt/core/maxwellcircuit.py

```diff
@@ -242,3 +242,53 @@ def export_netlist_from_schematic(self, output_file):
             return output_file
         except Exception:
             return False
+
+    def create_page(self, name):
+        """Add a new page to circuit schematic.
+
+        Parameters
+        ----------
+        name : str, int or float
+            Name to be used when creating the new circuit schematic.
+
+        Returns
+        -------
+        bool
+            ``True`` when page added, ``False`` when failed.
+
+        References
+        ----------
+        >>> oEditor.CreatePage()
+
+        Examples
+        --------
+        >>>from ansys.aedt.core import MaxwellCircuit
+        >>>app=MaxwellCircuit()
+        >>>component=app.create_page(name="page_name")
+        """
+        if not isinstance(name, (str, int, float)):
+            self.logger.error("Wrong type for argument ``name``.")
+            return False
+        self.oeditor.CreatePage(name)
+        return True
+
+    def get_num_pages(self):
```

Comment:
I think it would be better to add `nb_pages` as a property instead of adding this feature as a method.

Reply:
@SMoraisAnsys , That would be a better implementation. I'll make that switch to a parameter

Reply:
Not exactly, I had in mind:

```python
@property
def nb_pages(self):
    return self.oeditor.CreatePage(name)
```